flask_whooshee.py

```diff
@@ -156,8 +157,13 @@ def __init__(self, app=None):
     def init_app(self, app):
 
         self.index_path_root = app.config.get('WHOOSHEE_DIR', '') or 'whooshee'
-        self.search_string_min_len = app.config.get('WHOOSHEE_MIN_STRING_LEN', 3)
         self.writer_timeout = app.config.get('WHOOSHEE_WRITER_TIMEOUT', 2)
+        self.search_string_min_len = app.config.get('WHOOSHEE_MIN_STRING_LEN', 3)
+
+        if app.config.get('WHOOSHE_MIN_STRING_LEN', None) is not None:
+            warnings.warn(WhoosheeDeprecationWarning("The config key WHOOSHE_MIN_STRING_LEN has been renamed to WHOOSHEE_MIN_STRING_LEN. The mispelled config key is deprecated and will be removed in upcoming releases. Change it to WHOOSHEE_MIN_STRING_LEN to suppress this warning"))
+            self.search_string_min_len = ('WHOOSHE_MIN_STRING_LEN', 3)
```

Comment:
And since you know that the value actually exists, you can just use `self.search_string_min_len = app.config.get('WHOOSHE_MIN_STRING_LEN')` AFAICS.